src/schemas/fairdata-ida.ui.js

```diff
@@ -171,8 +171,8 @@ export default {
 		},
 		'placeholder': "– choose location –",
 		'label': "location",
-		'description': "This is some fancy optional description for the location field",
-		'help': "This is the optional help text for the location field",
+		'description': "Location for the dataset (YSO places), e.g. location of observations.",
```

Comment:
is this YSO or KOKO?

Reply:
My understanding is that KOKO includes YSO, but YSO places seems to be a distinct thing, so YSO places is most likely correct.